src/main/java/bh/bot/app/AbstractApplication.java

```diff
@@ -417,8 +421,7 @@ protected Point findImageBasedOnLastClick(BwMatrixMeta im) {
 						blackPixelDRgb, //
 						sc.getRGB(px[0], px[1]) & 0xFFFFFF, //
 						Configuration.Tolerant.color, //
-						im.getOriginalPixelPart(px[0], px[1])
-				)) {
+						im.getOriginalPixelPart(px[0], px[1]))) {
```

Comment:
Please less style-change like this next time, it's hard for review main logic

Reply:
my bad, that was my "format document" in VS Code. hit a lot more code than I thought it would